### ISSUE TITLE
edit/create 페이지에서 버튼 id 중복 관련

### ISSUE THREAD

**Issue by ooffordable** [swsnu/swppfall2018#102]
edit/create 페이지 requirement에서 write mode 와 preview mode에서 id가 중복되는 button들이 있습니다. 
아이디가 같기 때문에 하나의 button을 만드라는 뜻으로 이해했는데, 맞나요?

**Comment by sanha** [comment]
네, 지금 보시는 Github issue page의 comment 작성 뷰와 비슷하다고 생각하시면 됩니다. Write / Preview tab button이 있고 Close and comment / Comment 버튼이 있을 때 tab button들로 탭을 바꾸어도 Close and Comment 버튼과 Comment 버튼은 그대로 있습니다. 이와 유사하게 구현해주세요.

다만, #98 에도 언급된 것 처럼 comment에 대한 edit, delete 버튼은 같은 id를 갖는 button이 각 comment마다 있도록 구현해주세요. (id는 unique하도록 구현하는 것이 올바르나 미리 확인하지 못했습니다.)
